examples/neighbors/plot_nca_illustration.py

```diff
@@ -35,8 +35,7 @@
 # Draw the graph nodes
 for i in range(X.shape[0]):
     ax.text(X[i, 0], X[i, 1], str(i), va='center', ha='center')
-    ax.scatter(X[i, 0], X[i, 1], s=300, c=cm.Set1(y[i]), alpha=0.4)
-
+    ax.scatter(X[i, 0], X[i, 1], s=300, c=np.array([cm.Set1(y[i])]), alpha=0.4)
```

Comment:
The easier fix is to keep the 2D structure

```suggestion
    ax.scatter(X[i, 0], X[i, 1], s=300, c=cm.Set1(y[[i]])], alpha=0.4)
```

Reply:
so i shouldn't have changes it to an array?

Reply:
Try the fix and check the type and you will see that `y[[1]]` is an array ;)